### DISCUSSION TITLE
Cancelling subscriptions

### DISCUSSION THREAD

**Discussion by newtarkprojects** [gameserverapp/Platform#460] [Suggestions]
Hey Max,

It would be useful to have a 'are you sure you want to cancel' page when using the Subscriptions page under billing, after cancel has been clicked. I clicked cancel by accident while scrolling through my settings and had to go to add ons to set it up again. Would be useful anywhere else relating to cancelling GSA services.

Thank you,
Gem

**Comment by mvaessen** [comment]
Good point! i'll add this as a development item.
